# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/ActorsParent.cpp
# Commit: 307bd8d24407
# Full Hash: 307bd8d244079e2ee96773b93938f8d556041211
# Author: Cosmin Sabou <csabou@mozilla.com>
# Date: 2019-11-05 21:49:23
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Fix build bustages on ActorsParent.cpp. r=sgiesecke CLOSED TREE
# ==============================================================================

diff -r 09a93ed6c6f7 -r 307bd8d24407 dom/indexedDB/ActorsParent.cpp
--- a/dom/indexedDB/ActorsParent.cpp	Tue Nov 05 15:54:39 2019 +0200
+++ b/dom/indexedDB/ActorsParent.cpp	Tue Nov 05 16:15:53 2019 +0200
@@ -9389,6 +9389,7 @@
     case ComparisonOperator::GreaterOrEquals:
       return NS_LITERAL_CSTRING(">=");
   }
+  MOZ_CRASH("Should never get here.");
 }
 
 nsAutoCString GetKeyClause(const nsCString& aColumnName,
